File: backend/Multibanca.Domain.Models/Multibanca/recepcionar_matriz.cs
using Common.Domain.Models;
using System;

namespace Multibanca.Domain.Models.Multibanca
{
    public class recepcionar_matriz: base_auditoria
    {
        public int id_recepcionar_matriz { get; set; }
        public long id_expediente { get; set; }
        public string? observaciones { get; set; }
    }
}

File: frontend/src/features/actividades/recepcionar_matriz/models/recepcionar_matriz.ts
export interface RecepcionarMatriz {
  id_recepcionar_matriz: number;
  id_expediente: number;
  observaciones?: string | null;

  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}
